db/migrations/008-temp-insert-games.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO Games(SRId, GameName, ShortName)
+    VALUES    ('xkdk4g1m', 'Jak and Daxter: The Precursor Legacy', 'Jak1'),
+              ('ok6qlo1g', 'Jak II', 'Jak2'),
```

Comment:
this seems like a good idea to initialize the DB with this data.  Since we know what it is / it's static.  Though to my point above, probably not a good fit for the migration files (since the second time you'll get primary-key violations since the data is already inserted).

I think a decent workaround if we want to have this fixture data in the migration files is to use `INSERT OR IGNORE INTO...` and make the `SRId` fields `UNIQUE`.  That way things won't explode if the data already exists.  Same goes for the below files (with `Version` being the trickiest since the name probably isn't unique.  Might need to include the int id in the insert?)

Reply:
Those comments all sound good to me, I made all those fixes on y/start.
I totally forgot about deleting the database file, instead of using drops.
I'm fine with putting the inserts in a different folder, I just thought it might be nice for testing.

Reply:
Oh, should VersionName be UNIQUE? I missed that one. Or we can deal with it later, it is pretty annoying.